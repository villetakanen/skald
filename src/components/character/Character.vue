<template>
  <v-card>
    ...
  </v-card>
</template>
