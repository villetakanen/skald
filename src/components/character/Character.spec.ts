import { shallowMount } from '@vue/test-utils'
import Character from './Character.vue'

describe('Mounted App', () => {
  const wrapper = shallowMount(Character)
  it('renders correctly', () => {
    const wrapper = shallowMount(Character)
    expect(wrapper.element).toMatchSnapshot()
  })
})
