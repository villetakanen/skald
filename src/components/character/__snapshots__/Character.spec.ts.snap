// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mounted App renders correctly 1`] = `
<v-card-stub
  loaderheight="4"
  tag="div"
>
  
  ...

</v-card-stub>
`;
